server/src/test/scala/cromwell/jobstore/JobStoreServiceSpec.scala

```diff
@@ -19,53 +21,55 @@ import scala.concurrent.duration._
 
-      jobStoreService ! QueryJobCompletion(successKey, mockTask.outputs map WomMocks.mockOutputPort)
-      expectMsgType[JobNotComplete.type](MaxWait)
+        jobStoreService ! QueryJobCompletion(successKey, mockTask.outputs map WomMocks.mockOutputPort)
+        expectMsgType[JobNotComplete.type](MaxWait)
 
-      val outputs = WomMocks.mockOutputExpectations(Map("baz" -> WomString("qux")))
+        val outputs = WomMocks.mockOutputExpectations(Map("baz" -> WomString("qux")))
 
-      jobStoreService ! RegisterJobCompleted(successKey, JobResultSuccess(Option(0), outputs))
-      expectMsgType[JobStoreWriteSuccess](MaxWait)
+        jobStoreService ! RegisterJobCompleted(successKey, JobResultSuccess(Option(0), outputs))
+        expectMsgType[JobStoreWriteSuccess](MaxWait)
 
-      jobStoreService ! QueryJobCompletion(successKey, mockTask.outputs map WomMocks.mockOutputPort)
-      expectMsgPF(MaxWait) {
-        case JobComplete(JobResultSuccess(Some(0), os)) if os == outputs =>
-      }
+        jobStoreService ! QueryJobCompletion(successKey, mockTask.outputs map WomMocks.mockOutputPort)
+        expectMsgPF(MaxWait) {
+          case JobComplete(JobResultSuccess(Some(0), os)) if os == outputs =>
+        }
 
-      val failureCall = WomMocks.mockTaskCall(WomIdentifier("qux"))
-      val failureKey = BackendJobDescriptorKey(failureCall, None, 1).toJobStoreKey(workflowId)
+        val failureCall = WomMocks.mockTaskCall(WomIdentifier("qux"))
+        val failureKey = BackendJobDescriptorKey(failureCall, None, 1).toJobStoreKey(workflowId)
 
-      jobStoreService ! QueryJobCompletion(failureKey, mockTask.outputs map WomMocks.mockOutputPort)
-      expectMsgType[JobNotComplete.type](MaxWait)
+        jobStoreService ! QueryJobCompletion(failureKey, mockTask.outputs map WomMocks.mockOutputPort)
+        expectMsgType[JobNotComplete.type](MaxWait)
 
-      jobStoreService ! RegisterJobCompleted(failureKey, JobResultFailure(Option(11), new IllegalArgumentException("Insufficient funds"), retryable = false))
-      expectMsgType[JobStoreWriteSuccess](MaxWait)
+        jobStoreService ! RegisterJobCompleted(failureKey, JobResultFailure(Option(11), new IllegalArgumentException("Insufficient funds"), retryable = false))
+        expectMsgType[JobStoreWriteSuccess](MaxWait)
 
-      jobStoreService ! QueryJobCompletion(failureKey, mockTask.outputs map WomMocks.mockOutputPort)
-      expectMsgPF(MaxWait) {
-        case JobComplete(JobResultFailure(Some(11), _, false)) =>
-      }
+        jobStoreService ! QueryJobCompletion(failureKey, mockTask.outputs map WomMocks.mockOutputPort)
+        expectMsgPF(MaxWait) {
+          case JobComplete(JobResultFailure(Some(11), _, false)) =>
+        }
 
-      jobStoreService ! RegisterWorkflowCompleted(workflowId)
-      expectMsgType[JobStoreWriteSuccess](MaxWait)
+        jobStoreService ! RegisterWorkflowCompleted(workflowId)
+        expectMsgType[JobStoreWriteSuccess](MaxWait)
```

Comment:
This file appears to not actually have changed, other than some tabulation - is that correct?

Reply:
It's a bit hard to see but this test now requires a `SqlWorkflowStore` to work and that is implemented via a block that [indents everything one notch](https://github.com/broadinstitute/cromwell/pull/5906/files/5e8c6bf6d33f3f1bccc56ce069724343aff9cc06#diff-f3050b757f8fcd2049f131ce4bc27cf668ad824a93061100b5324c5c57a0b46fR32).

The underlying reason is that the `JobStoreActor` now [receives](https://github.com/broadinstitute/cromwell/pull/5906/files/5e8c6bf6d33f3f1bccc56ce069724343aff9cc06#diff-ae9a0cbfcd3ae4b785e4375decc6b8fa37b834e8920f694786dcfedccebfb156R20) a `WorkflowStoreAccess` in its `props`.